### PR TITLE
cleanup PFT, devops

### PR DESCRIPTION
**Pull request by george43g**
- cleanup code
- preparing for bigger PR

### CODE REVIEW

File: apps/backend-ops/src/emulator.ts [hunk 1]
```diff
@@ -116,7 +116,7 @@ export async function anonDbProcess() {
   console.info('Users synced!');
```
Comment (phpgeekfr) [inline apps/backend-ops/src/emulator.ts:116]: in `import { migrate, migrateBeta } from './migrations';`  migrate is unused

File: libs/firebase-utils/src/lib/firestore/emulator.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ function getFirestoreMetadataJsonFilename(firestorePath: string) {
  * This is a "default" path used by scripts and commands to store Firestore backups when running
  * various processes, such as PFT or db anonymization.
  */
-export const defaultEmulatorBackupPath = resolve(process.cwd(), '.firebase', 'firestore');
+export const defaultEmulatorBackupPath = join(process.cwd(), '.firebase', 'emulator');
```
Comment (phpgeekfr) [inline libs/firebase-utils/src/lib/firestore/emulator.ts:29]: in package.json we still have `--import ./.firebase/firestore`  This cannot cause a bug like we had last week ?

File: libs/firebase-utils/src/lib/firestore/emulator.ts [hunk 3]
```diff
@@ -160,33 +155,73 @@ export function connectEmulator(): FirestoreEmulator {
  * `localPath` is the local relative or absolute path where to find a FIRESTORE (not Firebase emulator) backup folder
  */
 export async function uploadDbBackupToBucket({ bucketName, remoteDir, localPath }: { bucketName: string; remoteDir?: string; localPath?: string; }) {
+  console.log('Uploading local Firestore backup to bucket. Converting from emulator format to Firestore format.');
+  console.log('Bucket:', bucketName);
+
   const firestoreExportAbsPath = localPath
-    ? join(process.cwd(), localPath)
+    ? resolve(localPath)
```
Comment (phpgeekfr) [inline libs/firebase-utils/src/lib/firestore/emulator.ts:162]: I remember that Francois said `resolve` can create issues on windows. It is ok to use it here ?

File: package.json [hunk 4]
```diff
@@ -155,6 +155,7 @@
     "bundlesize": "^0.18.0",
     "camelcase": "^6.1.0",
     "codelyzer": "6.0.1",
+    "commander": "^7.0.0",
```
Comment (phpgeekfr) [inline package.json:158]: Do you already use  `commander` ?

File: package.json [hunk 5]
```diff
@@ -164,6 +165,7 @@
     "firebase-functions-test": "^0.2.1",
     "firebase-tools": "^9.3.0",
     "husky": "^3.0.5",
+    "inquirer": "^7.3.3",
```
Comment (phpgeekfr) [inline package.json:168]: same

**Review by phpgeekfr — APPROVED** [review]

File: libs/firebase-utils/src/lib/firestore/emulator.ts (hunk 2 above)
Comment (george43g) [reply libs/firebase-utils/src/lib/firestore/emulator.ts:29]: we don't have this

File: package.json (hunk 4 above)
Comment (george43g) [reply package.json:158]: yes

File: package.json (hunk 5 above)
Comment (george43g) [reply package.json:168]: yes

File: libs/firebase-utils/src/lib/firestore/emulator.ts (hunk 2 above)
Comment (phpgeekfr) [reply libs/firebase-utils/src/lib/firestore/emulator.ts:29]: We do have it in `firebase:emulator` command

**Pull request merged**